app/ui-react/packages/ui/src/Integration/IntegrationStatus.tsx

```diff
@@ -30,24 +29,24 @@ export class IntegrationStatus extends React.Component<
         ? 'primary'
         : 'default';
     let label = PENDING; // it's a parachute
-    let style:any = { background: global_default_color_100.value };
+    let color: 'blue' | 'cyan' | 'green' | 'orange' | 'purple' | 'red' | 'grey' = 'grey';
     switch (this.props.currentState) {
       case PUBLISHED:
         label = this.props.i18nPublished;
-        style = { background: global_active_color_100.value };
+        color = 'blue';
         break;
       case UNPUBLISHED:
         label = this.props.i18nUnpublished;
         break;
       case ERROR:
         label = this.props.i18nError;
-        style = { background: global_danger_color_100 };
+        color = 'red';
```

Comment:
Are `blue` and `red` show up as in the palette we're using, or are these `#f00` and `#00f`?

Reply:
Yep they correspond the hex values defined in the PF color palette, they don't export that type that lists the colors so I just redefined it there in the component - not sure the specific hex values off the top of my head but I can take a look soon - totally customizable if we need btw, I just used these because it provided better contrast than the previous dark text against the darker background color.

Reply:
Ah, did those constants get removed then?

Reply:
No actually they are new! We didn't used to have all the fancy color support for the Label component in PF, and so lots of consumers were manually managing the background and text color which can be hit and miss - so I figured we could probably just use the built in color options for a more consistent look since there was concern about these labels and the text color being hard to read.

For example, 'blue' applies the css variable `--pf-c-label--m-blue--BackgroundColor` to the container - here's the [mapping](https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/components/Label/Label.tsx#L57-L65) for reference